### PR TITLE
Changing the name of function for sklearn compatibility.

### LINKED ISSUE #382: Skorch incompatibility, eeg augmentation
This error is a little confusing to explain, but I will try my best to explain it. 

Unfortunately, I need some help before submitting a pull request. I was trying to perform a parameter search with GridCVSearch following the methodology of the article developed by @cedricrommel and @jpaillard, available on his [page](https://cedricrommel.github.io/assets/pdfs/systematic_comparison.pdf).

I started with the data augmentation tutorial available in braindecode and used what is contained in the Pull Request number #349 to implement a parameter search using the skorch infrastructure. 

The first error that appeared is associated with the recently implemented transformation functions in braindecode and related to Skorch compatibility (commit #361).

To the best of our knowledge, for skorch to work correctly, all submodules inside the EEGClassifier must be able to receive a parameter (deep=True), or some kwargs, or inherit an estimator from sklearn. 

I consulted the skorch team [on this issue](https://github.com/skorch-dev/skorch/issues/794), and I debugged this error in more detail, and I arrived that the core issue is contained in the clone part: 

image

Following this, I tried to add a deep parameter to one of the transformations, such as the first picture below, and I got a different error message. Still, there is a new problem with redefining the augmentation functions.

image

image

Given this scenario, I believe there is some minor bug in skorch compatibility with the augmentation functions.

It is possible to work around this issue by performing cross-validation by hand and without the sklearn library. So I don't know how relevant this issue is.

### PR DESCRIPTION
**Pull request by bruAristimunha**
closes #382

As described in issue #382, a minor name change to allow hyperparameter search with GridsearchCV.

### CODE REVIEW

**Comment by agramfort** [comment]
@bruAristimunha can you document the API change in https://github.com/braindecode/braindecode/blob/master/docs/whats_new.rst ?

ok for you @robintibor @cedricrommel @gemeinl

**Review by gemeinl — APPROVED** [review]

**Comment by cedricrommel** [comment]
All seems good to me! Just don't get how come the coverage dropped... 🙃

**Comment by agramfort** [comment]
@bruAristimunha can you document the API change in https://github.com/braindecode/braindecode/blob/master/docs/whats_new.rst ?

File: docs/whats_new.rst [hunk 1]
```diff
@@ -31,6 +31,8 @@ Bugs
 API changes
 ~~~~~~~~~~~
 
+- Renaming the name of the function `get_params` to `get_augmentation_params` in :class:`braindecode.augmentation.base` and in all the function inside :class:`braindecode.augmentation.transforms`. Make Transform module cloneable for scikit-learn (:gh:`388` by `Bruno Aristimunha`_ and `Alex Gramfort`_)
```
Comment (agramfort) [inline docs/whats_new.rst:34]:
```suggestion
- Renaming the method `get_params` to `get_augmentation_params` in augmentation classes. This makes the Transform module compatible with scikit-learn cloning mechanism (:gh:`388` by `Bruno Aristimunha`_ and `Alex Gramfort`_)
```

**Review by agramfort — APPROVED** [review]

**Review by cedricrommel — APPROVED** [review]

**Comment by agramfort** [comment]
Thx @bruAristimunha

**Comment by bruAristimunha** [comment]
Happy to help!

**Comment by robintibor** [comment]
Great stuff thanks

**Pull request merged**